CHANGELOG.md

```diff
@@ -2,6 +2,22 @@
 
 ## 0.1 series
 
+### 0.1.9
+
+Library:
+
+* Allow URL as filenames for YAML files and SBML models (Closes #187) (#459)
+* Allow model time in observable formulas (#445)
+* Make float parsing from CSV round-trip (#444)
+* Validator: Error message for missing IDs, with line numbers. (#467)
+* Validator: detect duplicated observable IDs (#446)
```

Comment:
lower/upper case inconsistent :nerd_face:

Reply:
For consistency with the rest of the file... :innocent: